app/views/curation_concerns/file_sets/upload/_script_templates.html.erb

```diff
@@ -1,8 +1,7 @@
-<!-- TODO: make the upload_set_id available on the presenter -->
-<% if false && presenter.upload_set_id %>
+<% if @presenter && @presenter.upload_set_id %>
```

Comment:
In sufia 6, the files_controller_behavior.rb the `#new` action set:
```
@batch_id = ActiveFedora::Noid::Service.new.mint
```
you should just make the new action create a presenter that has a upload_set_id